account_financial_report/report/templates/aged_partner_balance.xml

```diff
@@ -114,18 +114,24 @@
                 <div class="act_as_cell" style="width: 32.52%;">Partner</div>
                 <!--## amount_residual-->
                 <div class="act_as_cell" style="width: 9.64%;">Residual</div>
-                <!--## current-->
-                <div class="act_as_cell" style="width: 9.64%;">Not due</div>
-                <!--## age_30_days-->
-                <div class="act_as_cell" style="width: 9.64%;">1 - 30 d.</div>
-                <!--## age_60_days-->
-                <div class="act_as_cell" style="width: 9.64%;">31 - 60 d.</div>
-                <!--## age_90_days-->
-                <div class="act_as_cell" style="width: 9.64%;">61 - 90 d.</div>
-                <!--## age_120_days-->
-                <div class="act_as_cell" style="width: 9.64%;">91 - 120 d.</div>
-                <!--## older-->
-                <div class="act_as_cell" style="width: 9.64%;">> 120 d.</div>
+                <t t-if="not age_partner_config">
```

Comment:
Here, it shouldn't be needed the `if`, as it should be already got in the Python structure.

Reply:
If it is not added, it will show all columns and it is not correct.